diff --git a/node_modules/@types/node-ipc/index.d.ts b/node_modules/@types/node-ipc/index.d.ts
index 5f16588..cc76968 100644
--- a/node_modules/@types/node-ipc/index.d.ts
+++ b/node_modules/@types/node-ipc/index.d.ts
@@ -196,6 +196,7 @@ declare namespace NodeIPC {
         emit(event: string, value: any): Client;
         emit(socket: Socket | SocketConfig, event: string, value?: any): Server;
         emit(socketConfig: Socket | SocketConfig, value?: any): Server;
+        broadcast(event: string, value: any): void;
     }
     interface SocketConfig {
         address?: string;
